components/cdc/Cargo.toml

```diff
@@ -61,6 +61,7 @@ txn_types = { path = "../txn_types" }
 concurrency_manager = { path = "../concurrency_manager" }
 fail = "0.4"
 lazy_static = "1.3"
+log_wrappers = { path = "../log_wrappers" }
```

Comment:
Why not remove hex dependency?

Reply:
I'll look through all modified crates and try removing them.

Reply:
I've removed unnecessary hex imports.